core/ide/src/main/java/org/alice/ide/custom/DoubleCustomExpressionCreatorComposite.java

```diff
@@ -77,4 +79,14 @@ protected String getTextForPreviousExpression(Expression expression) {
     }
     return text;
   }
+
+  @Override
+  protected Double getLastCustomValue() {
+    return ((DoubleLiteral) getValue()).value.getValue();
```

Comment:
Seeing this pattern repeated in each class makes me sad, but without reworking the generics in `CustomExpressionCreatorComposite` I guess this is what we have (and fulfills my earlier request).